modules/mcuboot/hooks/nrf53_hooks.c

```diff
@@ -15,6 +15,8 @@
 
 #define NET_CORE_SECONDARY_SLOT 1
 #define NET_CORE_VIRTUAL_PRIMARY_SLOT 3
+/* Hardcoded image size for network core slot 0. */
+#define NET_CORE_SLOT_0_IMG_SIZE (PM_MCUBOOT_PRIMARY_1_SIZE - PM_MCUBOOT_PAD_SIZE - 0x1000)
```

Comment:
What is 0x1000? Please provide a define to avoid magic number. Also, `PM_` defines are not "hardcoded" IMO. Couldn't you use `PM_CPUNET_APP_SIZE` ?

Reply:
The value seems "hardcoded" from user perspective, i.e. it does not reflect actual image size. Size of the partition is the same among all of the Thingy:5-compatible applications too.

Reply:
The 0x1000 was used to avoid letting image to occupy the whole image slot (that was a workaround for https://github.com/mcu-tools/mcuboot/pull/1140)

Reply:
The PM_CPUNET_APP_SIZE should work fine here. I will double check it and apply this one.

Reply:
Using PM_CPUNET_APP_SIZE  should work fine here.
It seems like there is a generic issue related partition sizes. We update only application on the network core and we reserve space for both the application and B0n bootloader which is quite misleading and leads to wasting memory.

Reply:
@hakonfam, why we define the slot sizes in that way ?

Reply:
Where do we reserve space for both the application and B0n bootloader? In the external flash secondary partition?

Reply:
It seems like it